### PR TITLE
add pyterrier (information retrieval) redirect message

### PR DESCRIPTION
**Pull request by cmacdonald**
Dear @dfurtado,

As discussed, please consider for your review, a PR for `init()`. A screenshot of it in use is at:
image

As per setup.py warning - I dont think its possible - see https://stackoverflow.com/a/44617404

### CODE REVIEW

File: pyterrier/__init__.py [hunk 1]
```diff
@@ -1,2 +1,7 @@
 from .http import http_result, view_result, Ok, NotFound, NoContent as http
 from .pyterrier import PyTerrier
+
+def init(*args, **kawrgs):
```
Comment (dfurtado) [inline pyterrier/__init__.py:4]: It seems to have a typo here, it should be `kwargs`

**Comment by cmacdonald** [comment]
Good spot. typo fixed.

File: pyterrier/__init__.py [hunk 2]
```diff
@@ -1,2 +1,7 @@
...
+  raise NotImplementedError("You have installed pyterrier the web framework, not PyTerrier the information retrieval toolkit - " +
```
Comment (dfurtado) [inline pyterrier/__init__.py:5]: I have a limit of 88 chars per line (default of black formatter).

You can perhaps change to something like this:
```python
def init(*args, **kwargs):
    raise NotImplementedError(
        (
            "You have installed pyterrier the web framework, not PyTerrier the "
            "information retrieval toolkit - To install the latter, use "
            "'pip install python-terrier'. For more information see "
            "https://github.com/terrier-org/pyterrier"
            "/wiki/Installation-of-Wrong-Package"
        )
    )
```
This way you don't need to concat the strings using `+` sign and you can keep the line to 88 chars.

Comment (dfurtado) [reply pyterrier/__init__.py:5]: Another thing, the indentation seems a bit off after the `def`, only 2 spaces.

Comment (cmacdonald) [reply pyterrier/__init__.py]: Sorry, should have written offline, rather than in Github editor. now fixed.

Comment (dfurtado) [reply pyterrier/__init__.py]: No worries. It still very long line 100+ chars. I will merge it anyway and I can fix that before I do a release. Would that be ok?

Comment (cmacdonald) [reply pyterrier/__init__.py]: > No worries. It still very long line 100+ chars.

Sorry, I seemed to have missed that comment during editing. 
> I will merge it anyway and I can fix that before I do a release. Would that be ok?

Sure, your package, your styling. We appreciate your help.

Comment (dfurtado) [reply pyterrier/__init__.py]: Cool, I will merge. I will add the fix today and release a new version on PyPI tonite. :)

**Comment by dfurtado** [comment]
Thanks for the fixed @cmacdonald !! I hope this fix will help your users and let me know if there is something more I can help with. 👍🏼

**Pull request merged**